commit 30e4352349623fbe02d2709e96bc3d2a93aafbab
Author: Henry Ptasinski <henryp@broadcom.com>
Date:   Thu Oct 21 11:04:11 2010 -0700

    staging: brcm80211: Remove unnecessary includes from bcmutils.c
    
    Removed includ of bcmip.h and vlan.h, as they're no longer needed.
    
    Signed-off-by: Henry Ptasinski <henryp@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/util/bcmutils.c
+++ b/drivers/staging/brcm80211/util/bcmutils.c
@@ -27,8 +27,6 @@
 #include <bcmendian.h>
 #include <bcmdevs.h>
 #include <proto/ethernet.h>
-#include <proto/vlan.h>
-#include <proto/bcmip.h>
 #include <proto/802.1d.h>
 #include <proto/802.11.h>
 
